eg/Classes/NetworkSend.py

```diff
@@ -23,6 +23,17 @@
 
 ENCODING = locale.getdefaultlocale()[1]
 
+
+def ShowError(msg):
+    app = wx.PyApp()
+    wx.MessageBox(
+        message=msg,
+        caption="EventGhost - warning",
+        style=wx.ICON_EXCLAMATION | wx.OK,
+        parent=None
+    )
+
```

Comment:
That is a good argument. We just need a MessageBox so there is no need to load full wx system. I will change it.